boards/arm/sama5/giant-board/src/giant-board.h

```diff
@@ -0,0 +1,436 @@
+ */
+
+#define PIO_LED_ORANGE (PIO_OUTPUT | PIO_CFG_DEFAULT | PIO_OUTPUT_SET | \
+                       PIO_PORT_PIOA | PIO_PIN6)
+
+/* SDMMC clocking
+ *
+ * Multimedia Card Interface clock (MCCK or MCI_CK) is Master Clock (MCK)
+ * divided by (2*(CLKDIV+1)).
+ *
+ *   MCI_SPEED = MCK / (2*(CLKDIV+1))
+ *   CLKDIV = MCI / MCI_SPEED / 2 - 1
+ *
+ * Where CLKDIV has a range of 0-255.
+ */
+
+/* MCK = 96MHz, CLKDIV = 119, MCI_SPEED = 96MHz / 2 * (119+1) = 400 KHz */
+
+#define SDMMC_INIT_CLKDIV          (119 << SDMMC_MR_CLKDIV_SHIFT)
+
+/* MCK = 96MHz, CLKDIV = 3, MCI_SPEED = 96MHz / 2 * (3+1) = 12 MHz */
+
+#define SDMMC_MMCXFR_CLKDIV        (3 << SDMMC_MR_CLKDIV_SHIFT)
+
+/* MCK = 96MHz, CLKDIV = 1, MCI_SPEED = 96MHz / 2 * (1+1) = 24 MHz */
+
+#define SDMMC_SDXFR_CLKDIV         (1 << SDMMC_MR_CLKDIV_SHIFT)
+#define SDMMC_SDWIDEXFR_CLKDIV     SDMMC_SDXFR_CLKDIV
+
+/* SDMMC Card Slots *********************************************************/
+
+/* The Giant Board provides a SD memory card slot:
+ *  a full size SD card slot (J6)
+ *
+ * The full size SD card slot connects via SDMMC1.  The card detect discrete
+ * is available on PA21 (pulled high) and shared with DAT3.  The write
+ * protect discrete is not connected and not available to software.  The
+ * slot only supports 4-bit wide transfer mode, and the NuttX driver
+ * currently uses only the 4-bit wide transfer mode.
+ *
+ *   PA18 SDMMC1_DAT0
+ *   PA19 SDMMC1_DAT1
+ *   PA20 SDMMC1_DAT2
+ *   PA21 SDMMC1_DAT3/SDMMC1_CD
+ *   PA22 SDMMC1_CK
+ *   PA28 SDMMC1_CMD
+ */
+
+#define IRQ_SDMMC1_CD   SAM_IRQ_PA30
+
+/* USB Ports ****************************************************************/
+
+/* The Giant Board features two USB communication ports:
+ *
+ *   1. Port A Host High Speed (EHCI) and Full Speed (OHCI) multiplexed with
+ *      USB Device High Speed Micro AB connector, J3
+ *
+ *   2. Port B Host High Speed (EHCI) and Full Speed (OHCI) High Speed Micro
+ *      AB connector, J7
+ *
+ */
+
+/****************************************************************************
+ * Public data
+ ****************************************************************************/
+
+#ifndef __ASSEMBLY__
+
+/****************************************************************************
+ * Public Function Prototypes
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: sam_bringup
+ *
+ * Description:
+ *   Bring up board features
+ *
+ ****************************************************************************/
+
+int sam_bringup(void);
+
+/****************************************************************************
+ * Name: sam_sdmmc_initialize
+ *
+ * Description:
+ *   Initialize and configure one SDMMC slot
+ *
+ ****************************************************************************/
+
+#ifdef HAVE_SDMMC
+int sam_sdmmc_initialize(int slotno, int minor);
+#endif
+
```

Comment:
This should be removed.  The the SDMMC (HSMCI) interface is not supported on the Giant Board.